index.html

```diff
@@ -2,37 +2,72 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Awesome Books</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome Books</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
 
-    <header>
-        <h1>All awesome books</h1>
-    </header>
 
-    <main>
+  <header class="navbar">
+    <h1 class="navHeading">Awesome books</h1>
+    <nav>
+      <ul class="links">
+        <li class="link"><a href="#header">List</a></li>
+        <li class="link"><a href="#new">Add new</a></li>
+        <li class="link"><a href="#contact">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
 
-        <div id="books"></div>
-        <hr class="book-underline">
-        <h2>Add a new book</h2>
-        <form>
-            <label for="title"></label>
-            <input type="text" id="title" placeholder="Title" required>
+  <section class="time">
+    <p id="time"></p>
+  </section>
 
-            <label for="author"></label>
-            <input type="text" id="author" placeholder="Author" required>
+  <main>
 
-            <button type="button" id="addButton">Add</button>
-        </form>
+    <section id="header">
+      <h1>All awesome books</h1>
+      <div id="books">
```

Comment:
- [optional] please, you can set the form to reset, so it clears it after submitting to stop a user from continuously clicking the submit button